---
title: BarCodeRecognitionException
second_title: Aspose.BarCode .NET API संदर्भ के लिए
description: क एक नय उदहरण प्ररंभ करत हैBarCodeRecognitionExceptionaspose.barcode.barcoderecognition/barcoderecognitionexception/ वर्ग.
type: docs
weight: 10
url: /hi/net/aspose.barcode.barcoderecognition/barcoderecognitionexception/barcoderecognitionexception/
---
## BarCodeRecognitionException() {#constructor}

का एक नया उदाहरण प्रारंभ करता है[`BarCodeRecognitionException`](../) वर्ग.

```csharp
public BarCodeRecognitionException()
```

### यह सभी देखें

* class [BarCodeRecognitionException](../)
* नाम स्थान [Aspose.BarCode.BarCodeRecognition](../../barcoderecognitionexception/)
* सभा [Aspose.BarCode](../../../)

---

## BarCodeRecognitionException(string) {#constructor_1}

का एक नया उदाहरण प्रारंभ करता है[`BarCodeRecognitionException`](../) वर्ग.

```csharp
public BarCodeRecognitionException(string message)
```

| पैरामीटर | प्रकार | विवरण |
| --- | --- | --- |
| message | String | त्रुटि संदेश |

### यह सभी देखें

* class [BarCodeRecognitionException](../)
* नाम स्थान [Aspose.BarCode.BarCodeRecognition](../../barcoderecognitionexception/)
* सभा [Aspose.BarCode](../../../)

---

## BarCodeRecognitionException(string, Exception) {#constructor_2}

का एक नया उदाहरण प्रारंभ करता है[`BarCodeRecognitionException`](../) वर्ग निर्दिष्ट त्रुटि संदेश और वर्तमान अपवाद के साथ।

```csharp
public BarCodeRecognitionException(string message, Exception innerException)
```

| पैरामीटर | प्रकार | विवरण |
| --- | --- | --- |
| message | String | अपवाद का त्रुटि संदेश। |
| innerException | Exception | वर्तमान अपवाद फेंक दिया गया है। |

### यह सभी देखें

* class [BarCodeRecognitionException](../)
* नाम स्थान [Aspose.BarCode.BarCodeRecognition](../../barcoderecognitionexception/)
* सभा [Aspose.BarCode](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.BarCode.dll -->
